src/freenet/node/updater/NodeUpdater.java

```diff
@@ -91,15 +91,23 @@ public abstract class NodeUpdater implements ClientGetCallback, USKCallback, Req
 	}
 
 	void start() {
+        try {
+            this.subscribe();
```

Comment:
The `this` qualifier is over-doing it a bit, methinks. 😄

Reply:
Oh yes — I blame Javascript ☺

Thank you!